README.md

```diff
@@ -124,3 +124,9 @@ During autoInstallBundle, all JS and SCSS are concatenated, minified, and instal
 3. In `ui.apps` run `gulp`.
 4. Start buidling great things in `ui.apps/src/main/resources/jcr_root/etc/slick/designs/...`
 5. You can also specify arguments when running gulp. Example: `$ gulp --slingHost='slick.millr.org' --slingPort=8181 --slingPass=MySuperPassword --slingUser=cmillar`
+
+# international characters support (UTF8 readiness)
+blogposts, comments and commentor names can have international characters
```

Comment:
I'd say move this into the feature list...

* International Language Support (UTF-8 Readiness)